huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_loadbalancer_test.go

```diff
@@ -453,3 +506,73 @@ resource "huaweicloud_elb_loadbalancer" "test" {
 }
 `, rName)
 }
+
+func testAccElbV3LoadBalancerConfig_availabilityZone(rName string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+data "huaweicloud_availability_zones" "test" {}
+
+resource "huaweicloud_vpc_subnet" "test_1" {
+  name       = "%[2]s_1"
+  vpc_id     = huaweicloud_vpc.test.id
+  cidr       = "192.168.1.0/24"
+  gateway_ip = "192.168.1.1"
+}
```

Comment:
it is useless, and fix the update method

Reply:
done